services/storage-users/pkg/config/config.go

```diff
@@ -59,7 +59,7 @@ type Log struct {
 
 // Service holds general service configuration
 type Service struct {
-	Name string `yaml:"-"`
+	Name string `yaml:"-" env:"STORAGE_USERS_SERVICE_NAME" desc:"Service name to use. Change this when starting an additional storage provider to prevent it from colliding with the default 'storage-users' service."`
```

Comment:
This is not clear as different terms are mixed. My understanding is, that we in general can start the service like with helm multiple times (it is not part of the "cant be scaleb by staring multiple times" list) - but with the same configuration. When configuring multiple storage providers, each provider needs a uniqe storage-users instance name. Correct? in case, we should rephrase to make it clear.